crates/biome_css_analyze/tests/specs/nursery/noDuplicateSelectors/valid.css.snap.new

```diff
@@ -5,56 +5,241 @@ expression: valid.css
 ---
 # Input
 ```css
-/* Same selector with different parent nodes are allowed */
-.foo {
-    width: 12;
-}
-
-@media (min-width: 10px) {
-    .foo {
-        width: 12;
-    }
-}
-
-.bar {
-    .bar {
-        width: 12;
-    }
-}
-
-/* Same selector in a different list */
-.foo .bar {
-    width: 12;
-}
-
-/* Net list is different */
-a b {
-    width: 12;
-}
-
-a {
-
-    & b,
-    & c {
-        width: 12;
-    }
-}
+/* no duplicates */
+a {} b {} c {} d, e, f {}
+
+/* duplicate inside media query */
+g {}
+@media print { g {} }	
+
+/* duplicate inside keyframes */
+@keyframes h { 0% {} }
+@keyframes h { 0% {} }
+
+/* duplicates inside nested rules */
+i { i { i {} } }
+
+/* selectors using parts of other selectors */
+.foo .bar {}
+.foo {}
+.bar {}
+.bar .foo {}
+
+/* selectors reused in other non-equivalent selector lists */
+j {} j, k {}
```

Comment:
Can you add `j {} k, j {}` too?
It matters especially if `k` happens not to be supported by a browser.
i.e. it may be on purpose